client/src/components/ForecastInfo.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import ContentContainer from "./ContentContainer";
+import styled from "@emotion/styled";
+
+const InfoTextContainer = styled.div`
+  padding: 0 2rem;
+`;
+
+export const ForecastInfo = () => {
+  return (
+    <ContentContainer>
+      <h1>Ökostrom</h1>
+      <InfoTextContainer>
```

Comment:
Looks like you create `InfoTextContainer` to add a padding. 
You could use `p` element here, which comes with a padding by default.